src/drivers/px4fmu/fmu.cpp

```diff
@@ -1026,74 +1026,75 @@ PX4FMU::cycle()
 			}
 		}
 
-		/* can we mix? */
-		if (_mixers != nullptr) {
+	}
 
-			size_t num_outputs;
+	_cycle_timestamp = hrt_absolute_time();
```

Comment:
Did `_cycle_timestamp` not get updated in the RC lost case? If so that's a good catch!